src/main/java/net/tardis/mod/client/guis/GuiOptifineWarning.java

```diff
@@ -11,32 +11,31 @@
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.relauncher.Side;
+import net.tardis.mod.config.TardisConfig;
 
 import java.io.IOException;
 import java.util.List;
 
+import javax.lang.model.util.ElementScanner6;
+
 public class GuiOptifineWarning extends GuiScreen {
 	
 	public static GuiScreen lastGui;
 
 	public GuiOptifineWarning() {
 	}
-	
+
 	@Override
 	public void initGui() {
-		super.initGui();
-		
-		this.addButton(new GuiButtonExt(0, this.width / 2 - 154, this.height - 48, 150, 20, I18n.format("Close Game")));
```

Comment:
Why has the close game button been removed?

Reply:
I hit it sometimes when clicking on the window and there's no point of it as people can just hit the normal games quit.